doc/tools.rst

```diff
@@ -36,16 +36,13 @@ PyMODM
    documentation is available on `readthedocs
    <https://pymodm.readthedocs.io/en/stable/>`_.
```

Comment:
Please move pymodm to the unmaintained list too.

Reply:
Done